doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1401,6 +1401,15 @@ Serial LTE Modem
 
 The issues in this section are related to the :ref:`serial_lte_modem` application.
 
+.. rst-class:: v2-5-0
+
+NCSDK-24757: SLM triggers indication PIN immediately after #XSLEEP=2.
+  When #XSLEEP=2 disables the UART, it writes "Ready" to the UART TX buffer, which immediately triggers the indication PIN, even though there is no incoming traffic or notifications.
```

Comment:
```suggestion
  When the ``#XSLEEP=2`` AT command disables the UART, it writes "Ready" to the UART TX buffer.
  This immediately triggers the indication PIN, even though there is no incoming traffic or notifications.
```
Should "Ready" be ``Ready``? Not sure on that one.

Reply:
It is plain Ready in the buffer, so perhaps `Ready`.